docs/en/appium-setup/running-on-osx.md

```diff
@@ -8,8 +8,8 @@ Appium on OS X supports iOS and Android testing.
 * Make sure you have Xcode and the iOS SDK(s) installed. Xcode version 7.1 is
   recommended as earlier versions of Xcode are limited in which versions of iOS
   they can test against. See the next section for more detail.
-* You need to authorize use of the iOS Simulator by running `sudo authorize_ios`
-  (`authorize_ios` is a binary made available by the Appium npm package). If you
+* You need to authorize use of the iOS Simulator by running `sudo node_modules/.bin/authorize-ios`
```

Comment:
`$(npm bin) won't work either for global install. I'm fine with just saying install globally, then run `authorize-ios`.